tests/functional/api/BackbeatServer.js

```diff
@@ -144,7 +143,7 @@ describe('Backbeat Server', () => {
         });
 
         after(() => {
-            redis.keys('test:bb:*').then(keys => {
+            redis.keys('*:test:bb:*').then(keys => {
```

Comment:
Is this intentional? Looks like the test keys (for example, `test:bb:ops`) would have been matched.

Reply:
Ooo, nice catch, this won't impact anything, but will push up a fix and rebase soon